config/bundlesize.js

```diff
@@ -4,12 +4,12 @@ module.exports = {
   app: {
     javascript: {
       pattern: 'assets/*.js',
-      limit: '375KB',
+      limit: '376KB',
       compression: 'gzip'
     },
     css: {
       pattern: 'assets/*.css',
-      limit: '22KB',
+      limit: '15KB',
```

Comment:
#205 reduced size of CSS assets by nearly 1/3 (7 KB) after gzip. Done very well @sappor0! :clap: